data/releases/schedule.yaml

```diff
@@ -114,12 +114,15 @@ schedules:
   maintenanceModeStartDate: 2022-08-28
   endOfLifeDate: 2022-10-28
   next:
-    release: 1.22.16
-    cherryPickDeadline: 2022-10-07
-    targetDate: 2022-10-12
+    release: 1.22.17
+    cherryPickDeadline: 2022-12-02
+    targetDate: 2022-12-07
     note: >-
-      The 1.22 release is in maintenance mode. As per the support policy, 1.22.16 will be released **only** if there are critical and/or security issues.
+      As per our support policy, the 1.22 release has reached End-of-Life (EOL), but we are planning to cut one more 1.22 patch release to migrate to the new `registry.k8s.io` image registry. Other changes will not be permitted.
```

Comment:
After that release is cut, change "we are planning" to reflect the new tense of the situation (ie, that it has happened) - or make some other change to cover this.

Would we still release 1.22.18 if there is a critical security issue?

Reply:
maybe not, i would say only if is related to the registry topic. otherwise after this patch the 1.22 will be EOL